examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
+	console.log('Logged into Steam');
 });
 
-client.on('webSession', function(sessionID, cookies) {
-	manager.setCookies(cookies, function(err) {
```

Comment:
@DoctorMcKay I mean, in almost each case (if not for all cases) where the the error-first callback is used, there is the error checking (and there should be, I guess).
In cases there is no error checking (I can't remember of these cases in this repo, but anyway) we can implement it later and it would be more clear what error we're checking.
Either way it'd benefit us.